projects/com.oracle.truffle.llvm.parser/src/com/oracle/truffle/llvm/parser/listeners/Function.java

```diff
@@ -982,6 +982,20 @@ private static int getAlign(long argument) {
         return (int) argument & (Long.SIZE - 1);
     }
 
+    private static Type getExtractedElementType(Type type) {
+        Type valType = type;
+        if (valType instanceof PointerType) {
```

Comment:
I fixed this in #742. Forward referenced symbols are not the common case and the instructions this error appeared in are not the most common either, so it is entirely possible for this to go unnoticed.